examples/tutorial_2_customproto.md

```diff
@@ -22,27 +22,33 @@ type TestProto = ref object of LPProtocol
 
 We've set a [protocol ID](https://docs.libp2p.io/concepts/protocols/#protocol-ids), and created a custom `LPProtocol`. In a more complex protocol, we could use this structure to store interesting variables.
 
-A protocol generally has two part: and handling/server part, and a dialing/client part.  
-Theses two parts can be identical, but in our trivial protocol, the server will wait for a message from the client, and the client will send a message, so we have to handle the two cases separately.
+A protocol generally has two parts: a handling/server part, and a dialing/client part. These two parts can be identical, but in our trivial protocol, the server will wait for a message from the client, and the client will send a message, so we have to handle the two cases separately.
 
-Let's start with the server part:
+Lets start with the server part:
```

Comment:
Is it Lets?

Reply:
Nope, you were definitely right! Really sorry for that.